app/services/big_blue_button_api.rb

```diff
@@ -12,13 +12,57 @@ def bbb_server
     @bbb_server ||= BigBlueButton::BigBlueButtonApi.new(bbb_endpoint, bbb_secret, '1.8')
   end
 
+  # Start a session for a specific room and returns the join URL.
+  def start_session(room:, session_starter:, options: {})
+    # TODO: amir - Revisit this.
+    create_options = default_create_opts.merge(options)
+    join_options = default_join_opts.merge(create_options) # TODO: amir - Revisit this (createTime,...).
+
+    meeting_id = room.friendly_id
+    user_name = session_starter&.name || 'Someone'
+    password = (session_starter && create_options[:moderatorPW]) || create_options[:attendeePW]
+
+    create_session meeting_name: room.name, meeting_id:, options: create_options
+    get_session_join_url meeting_id:, user_name:, password:, options: join_options
+  end
+
   private
 
+  def create_session(meeting_name:, meeting_id:, options: {}, modules: nil)
+    bbb_server.create_meeting meeting_name, meeting_id, options, modules
+  end
```

Comment:
I wouldn't break this off into another method - just add more complexity

Reply:
Alright, I'll remove it

Reply:
DONE